src/pages/Governance/Stake/Index.tsx

```diff
@@ -40,6 +42,7 @@ export function StakePage() {
     const isVoterAddrValid = validateVoterAddressInput();
 
     if (isStakingAmountValid && isOperatorAddrValid && isVoterAddrValid) {
+      setLoadingModalIsOpen(true);
```

Comment:
would it make more sense to expose those functions/state from the `useSubmitTransaction` file? so it would live in one file?